README.md

```diff
@@ -68,8 +68,15 @@ underscores is hard. So for example, `add_repo` can be typed as `addrepo`.
                                 name given in the second parameter. The location
                                 of the repo will be asked for and then added to
                                 your list of repos.
+* `repo_add_directory` or `rad` - Searches all subdirectories of the second parameter
+                                  and automatically adds each to the list of repos. This
```

Comment:
Doesn't it only add subdirectories with a .git directory in them? If that is the case, this should be updated to reflect that. If it isn't the case, I think it should be.